da/celestia/celestia.go

```diff
@@ -99,7 +97,7 @@ func (c *DataAvailabilityLayerClient) SubmitBlocks(ctx context.Context, blocks [
 		blobs[blockIndex] = blockBlob
 	}
 
-	txResponse, err := c.rpc.State.SubmitPayForBlob(ctx, math.NewInt(c.config.Fee), c.config.GasLimit, blobs)
+	DAHeightIncludedAt, err := c.rpc.Blob.Submit(ctx, blobs)
```

Comment:
```suggestion
	dataLayerHeight, err := c.rpc.Blob.Submit(ctx, blobs)
```

we're calling this `dataLayerHeight` in `RetrieveBlocks`, would be consistent.